src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -108,6 +108,11 @@ internal sealed class MemberExpansion : Expansion
                 expansions.Add(staticMembersExpansion);
             }
 
+            if (value.NativeComPointer != 0)
```

Comment:
"Native View" did not exist in the legacy EE.  Before non-public members seemed like a reasonable place to put it.  I don't mind moving it if you disagree.